src/ui/pygame.textinput.py

```diff
@@ -0,0 +1,184 @@
+"""
```

Comment:
Not gonna search deeply right now but where do we use this file at all?

Reply:
this is for creating a text input box
will be used for:
entering player name
some settings